client/src/worldgen.rs

```diff
@@ -133,13 +134,14 @@ pub fn voxels(graph: &DualGraph, node: NodeId, chunk: Vertex, dimension: u8) ->
     // empirically.
     const ELEVATION_MARGIN: f64 = 0.7;
     let center_elevation = state.surface.elevation(na::Vector3::repeat(0.5), chunk);
-    if center_elevation - ELEVATION_MARGIN > me_max / 10.0 {
+    let threshold = 10.0;
```

Comment:
This is a scaling factor, not a threshold. Also should probably be a `const`.

Reply:
Made the suggested changes.